mix.exs

```diff
@@ -31,9 +33,9 @@ defmodule PlugCanonicalHost.Mixfile do
   defp deps do
     [
       {:plug, " ~> 1.0"},
-      {:credo, "~> 0.6", only: :dev},
-      {:earmark, "~> 1.2", only: :dev},
-      {:ex_doc, "~> 0.19", only: :dev},
+      {:credo, "~> 0.10", only: :dev},
+      {:earmark, "~> 1.4", only: :dev},
+      {:ex_doc, "~> 0.23", only: :dev},
```

Comment:
I think I’d avoid upgrading these deps in this PR so its only purpose is fixing the Elixir 1.11 warning.

Reply:
> I think I’d avoid upgrading these deps in this PR so its only purpose is fixing the Elixir 1.11 warning.

It actually fixes other warnings. I could extract it to a separate PR though…

Reply:
If it fixes a few warnings as well, let’s keep them! 😄